dist/ckeditor.js

```diff
@@ -0,0 +1,6 @@
+/*!
```

Comment:
Should we provide `dist` directory in the github repo?

Reply:
Why not? https://github.com/ckeditor/ckeditor5-build-classic/tree/master/build

Reply:
So you'll be updating the `dist` directory every CKE5 release? 😄

Reply:
For what? If the source will change, the dist will be updated. CKE5 releases do not touch the package.